### PR TITLE
Add CentOS Stream 9

### PR DESCRIPTION
**Pull request by AlexBaranowski**
Hi,

I want to propose these changes to add CentOS Stream 9.

When it comes to GitLab links, I tried with a few packages, and it worked well.

Best,
Alex

### CODE REVIEW

File: repos.d/rpm/centos.yaml [hunk 1]
```diff
@@ -119,6 +119,35 @@
...
+  repolinks:
+    - desc: CentOS home
+      url: https://www.centos.org/
+  packagelinks:
+    - type: PACKAGE_SOURCES
+      url: 'https://gitlab.com/redhat/centos-stream/rpms/{srcname}/-/tree/c9s'
+    - type: PACKAGE_RECIPE
+      url: 'https://gitlab.com/redhat/centos-stream/rpms/{srcname}/-/blob/c9s/{srcname}.spec'
+    - type: PACKAGE_RECIPE_RAW
+      url: 'https://gitlab.com/redhat/centos-stream/rpms/{srcname}/-/blob/c9s/{srcname}.spec'
```
Comment (Conan-Kudo) [inline repos.d/rpm/centos.yaml:148]: This should be changed like so:

```suggestion
      url: 'https://gitlab.com/redhat/centos-stream/rpms/{srcname}/-/raw/c9s/{srcname}.spec'
```

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

File: repos.d/rpm/centos.yaml (hunk 1 above)
Comment (AlexBaranowski) [reply repos.d/rpm/centos.yaml]: @Conan-Kudo  You are right! I missed it. Nice catch.

**Comment by AMDmi3** [comment]
Is it centos stream 9 or centos 9 stream? Cause if the former is the case we probably need to rename centos 8 stream.

**Comment by Conan-Kudo** [comment]
The official name is "CentOS Stream 9". The older major version is "CentOS Stream 8".

**Comment by AlexBaranowski** [comment]
@AMDmi3 @Conan-Kudo I changed repo name and description for CentOS Stream 8

**Comment by AMDmi3** [comment]
Merged, thanks!

**Pull request closed without merging**